x-pack/plugins/apm/server/routes/metrics/by_agent/shared/memory/index.ts

```diff
@@ -60,6 +69,23 @@ export const percentSystemMemoryUsedScript = {
   `,
 } as const;
 
+export const cgroupMemoryFilter = {
+  bool: {
+    should: [
+      {
+        bool: {
+          filter: [
+            { exists: { field: METRIC_CGROUP_MEMORY_USAGE_BYTES } },
+            { exists: { field: METRIC_CGROUP_MEMORY_LIMIT_BYTES } },
+            { exists: { field: METRIC_SYSTEM_TOTAL_MEMORY } },
+          ],
+        },
+      },
+    ],
+    minimum_should_match: 1,
+  },
+};
```

Comment:
Shouldn't it be like this?

```suggestion
export const cgroupMemoryFilter = {
  bool: {
    should: [
      { exists: { field: METRIC_CGROUP_MEMORY_USAGE_BYTES } },
      { exists: { field: METRIC_CGROUP_MEMORY_LIMIT_BYTES } },
      { exists: { field: METRIC_SYSTEM_TOTAL_MEMORY } },
    ],
    minimum_should_match: 1,
  },
};
```